commcare_export/cli.py

```diff
@@ -203,9 +203,12 @@ def main_with_args(args):
 
     checkpoint_manager = None
     if writer.support_checkpoints:
-        checkpoint_manager = CheckpointManager(args.output, args.query, misc.digest_file(args.query))
-        with checkpoint_manager:
-            checkpoint_manager.create_checkpoint_table()
+        if not os.path.exists(args.query):
```

Comment:
If it's possible the path doesn't exist, is it possible it's empty?

Reply:
This got removed later but if it were empty then we'd get an error earlier on